main/src/main/res/values/strings.xml

```diff
@@ -7,79 +7,79 @@
-    <string name="clear_log">clear log</string>
+    <string name="clear_log">Clear log</string>
     <string name="title_cancel">Cancel Confirmation</string>
     <string name="cancel_connection_query">Disconnect the connected VPN/cancel the connection attempt?</string>
     <string name="remove_vpn">Remove VPN</string>
     <string name="check_remote_tlscert">Checks whether the server uses a certificate with TLS Server extensions (--remote-cert-tls server)</string>
     <string name="check_remote_tlscert_title">Expect TLS server certificate</string>
-    <string name="remote_tlscn_check_summary">Checks the Remote Server Certificate Subject DN</string>
+    <string name="remote_tlscn_check_summary">Verifies the subject DN of the remote server certificate</string>
     <string name="remote_tlscn_check_title">Certificate Hostname Check</string>
-    <string name="enter_tlscn_dialog">Specify the check used to verify the remote certificate DN (e.g. C=DE, L=Paderborn, OU=Avian IP Carriers, CN=openvpn.blinkt.de)\n\nSpecify the complete DN or the RDN (openvpn.blinkt.de in the example) or an RDN prefix for verification.\n\nWhen using RDN prefix \"Server\" matches \"Server-1\" and \"Server-2\"\n\nLeaving the text field empty will check the RDN against the server hostname.\n\nFor more details see the OpenVPN 2.3.1+ manpage under —verify-x509-name</string>
+    <string name="enter_tlscn_dialog">Specify the check used to verify the remote certificate DN (e.g. C=DE, L=Paderborn, OU=Avian IP Carriers, CN=openvpn.blinkt.de)\n\nSpecify the complete DN or the RDN (openvpn.blinkt.de in the example) or an RDN prefix for verification.\n\nWhen using RDN prefix \"Server\" matches \"Server-1\" and \"Server-2\"\n\nLeaving the text field empty will check the RDN against the server hostname.\n\nMore details in the OpenVPN 2.3.1+ manpage under —verify-x509-name</string>
     <string name="enter_tlscn_title">Remote certificate subject</string>
-    <string name="tls_key_auth">Enables the TLS Key Authentication</string>
+    <string name="tls_key_auth">Turns on TLS Key authentication</string>
     <string name="tls_auth_file">TLS Auth File</string>
     <string name="pull_on_summary">Requests IP addresses, routes and timing options from the server.</string>
-    <string name="pull_off_summary">No information is requested from the server. Settings need to be specified below.</string>
+    <string name="pull_off_summary">No info was requested from the server. Specify settings below.</string>
     <string name="use_pull">Pull Settings</string>
     <string name="dns">DNS</string>
-    <string name="override_dns">Override DNS Settings by Server</string>
-    <string name="dns_override_summary">Use your own DNS Servers</string>
+    <string name="override_dns">Override DNS Settings by server</string>
+    <string name="dns_override_summary">Use your own DNS servers</string>
     <string name="searchdomain">searchDomain</string>
```

Comment:
What is this? Domain to search in?

Reply:
searchDomain is basically a fixed term related to DNS configuration